velox/functions/prestosql/tests/ArraySortTest.cpp

```diff
@@ -583,6 +583,22 @@ TEST_F(ArraySortTest, lambda) {
       "(x, y) -> if(length(x) < length(y), 1, if(length(x) = length(y), 0, -1))");
 }
 
+TEST_F(ArraySortTest, failOnMapTypeSort) {
+  auto data = makeRowVector({makeDataArray<TestMapType>({2, 1, 0})});
```

Comment:
we can just use makeAllNullMapVector(10, INTEGER(), VARCHAR())

Reply:
Got it, thanks.